hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/LockManager.java

```diff
@@ -64,13 +69,18 @@ public void lock() {
       boolean acquired = false;
       while (retryCount <= maxRetries) {
         try {
+          metrics.startLockApiTimerContext();
           acquired = lockProvider.tryLock(writeConfig.getLockAcquireWaitTimeoutInMs(), TimeUnit.MILLISECONDS);
           if (acquired) {
+            metrics.updateLockAcquiredMetric();
+            metrics.startLockHeldTimerContext();
```

Comment:
can we combine both these into single method. we don't have any caller which calls either of these individually right?

Reply:
"updateLockAcquiredMetric" has a parallel function also "updateLockNotAcquiredMetricis" is the reason I have kept them separate.

Reply:
+1 on combining them.